packages/jest-resolve/src/defaultResolver.ts

```diff
@@ -241,19 +233,17 @@ function getPathInModule(
 
 function createResolveOptions(
   conditions: Array<string> | undefined,
-): ResolveExportsOptions {
+): resolve.Options {
   return conditions
     ? {conditions, unsafe: true}
     : // no conditions were passed - let's assume this is Jest internal and it should be `require`
       {browser: false, require: true};
 }
 
-function createImportsResolveOptions(conditions: Array<string> | undefined) {
-  return {
-    conditions: conditions
-      ? [...conditions, 'default']
-      : ['node', 'require', 'default'],
-  };
+function createImportsResolveOptions(
```

Comment:
so we still need separate options for `exports` and `imports`, or could `createResolveOptions` and `createImportsResolveOptions ` be the same?

Reply:
Yes they could be merged. Didn't know if you wanted to keep the two named helpers for differences later

Reply:
Nah, a single helper is fine 👍